src/FormMigration/FormMetaDecorator.php

```diff
@@ -631,23 +631,32 @@ public function getFundsAndDesignationsAttributes(): array
 
 
         $options = [];
-        foreach ($donorOptions as $fundId) {
-            $options[] = [
-                'value' => $fundId,
-                'label' => $this->getFundLabel($fundId),
-                'checked' => $isAdminChoice ? $fundId === $adminChoice : true,
-                'isDefault' => $this->isDefaultFund($fundId),
-            ];
+        if (!empty($donorOptions)) {
```

Comment:
@glaubersilva heads up, I encountered an issue with the funds step during my tests and had to modify it slightly.  Basically the logic was trying to reach into the database prematurely so i added extra conditionals.  I would appreciate if you could take a quick look.

Reply:
Cool, nice catch! So just checking if the options are not empty to prevent this problem, is that?

Let's add an unreleased tag in the getFundsAndDesignationsAttributes() informing about these changes.

Reply:
Resolved a8fb3d88